<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditMSidedCardTemplateWdgt</class>
 <widget class="QWidget" name="EditMSidedCardTemplateWdgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>484</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Front template:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="front_template"/>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label1_2">
           <property name="text">
            <string>→</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Styling (shared between sister cards):</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="css"/>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Preferred</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>13</width>
               <height>22</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="labelc1">
             <property name="text">
              <string>↗</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>13</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="labelc2">
             <property name="text">
              <string>↘</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Preferred</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>13</width>
               <height>5</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Back template:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="back_template"/>
           </item>
          </layout>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label1_3">
           <property name="text">
            <string>→</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Front preview:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWebEngineView2" name="front_preview" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>295</width>
             <height>50</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
           </property>
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
           <property name="url" stdset="0">
            <url>
             <string>about:blank</string>
            </url>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Back preview:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWebEngineView2" name="back_preview" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>295</width>
             <height>50</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
           </property>
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
           <property name="url" stdset="0">
            <url>
             <string>about:blank</string>
            </url>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView2</class>
   <extends>QWidget</extends>
   <header>mnemosyne/pyqt_ui/qwebengineview2</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../../anki/designer/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>front_template</sender>
   <signal>textChanged()</signal>
   <receiver>EditMSidedCardTemplateWdgt</receiver>
   <slot>update_preview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>css</sender>
   <signal>textChanged()</signal>
   <receiver>EditMSidedCardTemplateWdgt</receiver>
   <slot>update_preview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>back_template</sender>
   <signal>textChanged()</signal>
   <receiver>EditMSidedCardTemplateWdgt</receiver>
   <slot>update_preview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>406</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
